### PR TITLE
Hw3

### PR DESCRIPTION
**Pull request by gbielanova**
Functionality can be seen at https://gbielanova.github.io/beetroot_hws_js/hw3.html

Please note, inserting multiline scripts inside HTML is done for learning purposes and I'm getting that it's not the best option in real-life projects. Also, all functions have not self-explanatory names that are also done for simplify checking of work result and shouldn't be used in real projects.

### CODE REVIEW

**Comment by roskyba** [comment]
Got it. Thanks

File: js/hw3.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+function task1(num1, num2) {
+    return num1 < num2 ? -1 : num1 > num2 ? 1 : 0;
+}
+
+function task2(num) {
+    if (num === 0)
+        return 1;
+    else {
+        return num * task2(num - 1);
```
Comment (roskyba) [inline js/hw3.js:26]: Awesome. You've tried recursion

File: js/hw3.js [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+    (min == '') ? min = '00' : min;
+    (sec == '') ? sec = '00' : sec;
+    return `${hh}:${min}:${sec}`;
+}
+
+function task8(hh, min, sec) {
+    return Number(hh) * 60 * 60 + Number(min) * 60 + Number(sec);
+}
+
+function task9(sec) {
```
Comment (roskyba) [inline js/hw3.js:73]: 3600 will return "1:0:0". Which should be "1:00:00"

Comment (gbielanova) [reply js/hw3.js:73]: fixed

**Review by roskyba — APPROVED** [review]

**Pull request merged**